src/Generic/Data/Surgery/Internal.hs

```diff
@@ -866,6 +953,80 @@ type Arborify' arb op n nDiv2 f g =
    <=< SplitAt nDiv2
    ) (op f g)
 
+class GLazify f where
+  gLazify :: f x -> Lazify f x
```

Comment:
`gLazify` and `gUnLazify` both simplify to `coerce`.

Reply:
You're right!